src/App.jsx

```diff
@@ -88,6 +97,20 @@ function App() {
     setIsStatisic(false);
   };
 
+  const handleFetch = async () => {
+    try {
+      setLoading(true);
+      const res = await axios.get(`${API_URL}&token=${token}`);
+      setData(res.data.results);
+      setTotalQuestions((total) => total + 1);
```

Comment:
Increment the `total` counter only after the user answers the question.

This way you will avoid the problem of `0/1` on the first question, before the user has given their first reply.